server.ts

```diff
@@ -55,13 +55,15 @@ async function main() {
   const SHOULD_UPDATE_EVENTS = process.env.UPDATE_EVENTS === 'true';
   const SHOULD_UPDATE_BALANCES = process.env.UPDATE_BALANCES === 'true';
   const SHOULD_UPDATE_EDGEWARE_LOCKDROP_STATS = process.env.UPDATE_EDGEWARE_LOCKDROP_STATS === 'true';
+  const SHOULD_LOAD_DATABASE_FROM_TOKEN_LISTS = process.env.SHOULD_LOAD_DATABASE_FROM_TOKEN_LISTS === 'true';
```

Comment:
Should add a `yarn` command to the package.json that triggers this automatically.